tests/forte/data/data_store_test.py

```diff
@@ -642,6 +642,66 @@ def test_add_annotation_raw(self):
             ],
         )
 
+    def test_add_audio_annotation_raw(self):
+        # test add Document entry
+        tid_recording: int = self.data_store.add_audio_annotation_raw(
+            "ft.onto.base_ontology.Recording", 1, 5
+        )
+        # test add Sentence entry
+        tid_audio_utterance: int = self.data_store.add_audio_annotation_raw(
+            "ft.onto.base_ontology.AudioUtterance", 5, 8
+        )
+        tid_utterance: int = self.data_store.add_annotation_raw(
+            "ft.onto.base_ontology.Utterance", 5, 8
+        )
+        num_doc = len(
```

Comment:
these varaible names do not match the entry type. not that this is wrong, but a bit confusing to maintain, probably:

```suggestion
        num_recording = len(
```

here instead.